src/components/FileList/FileList.js

```diff
@@ -1,275 +1,215 @@
-// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
-// SPDX-License-Identifier: MIT
-
 import React, { Component } from 'react'
-import PropTypes from 'prop-types'
-import ReactTable from 'react-table'
-import 'react-table/react-table.css'
-import transform from 'lodash/transform'
-import isEqual from 'lodash/isEqual'
+import { Table, Input, Button, Icon } from 'antd'
 import get from 'lodash/get'
-import treeTableHOC from './treeTable'
-import FilterCustomComponent from './FilterCustomComponent'
-import FacetsRenderer from '../FacetsRenderer'
-import LicensesRenderer from '../LicensesRenderer'
-import CopyrightsRenderer from '../CopyrightsRenderer'
+import isArray from 'lodash/isArray'
+import CopyrightsRenderer from '../../components/CopyrightsRenderer'
+import LicensesRenderer from '../../components/LicensesRenderer'
+import FacetsRenderer from '../../components/FacetsRenderer'
 import Contribution from '../../utils/contribution'
 import FileListSpec from '../../utils/filelist'
-// Import Custom TreeTable HOC
-const TreeTable = treeTableHOC(ReactTable)
-
-/**
- * A File List Tree-view, according to https://github.com/clearlydefined/website/issues/191
- *
- */
 export default class FileList extends Component {
-  static propTypes = {
-    readOnly: PropTypes.bool
+  state = {
+    files: [],
+    filteredFiles: [],
+    filteredInfo: {},
+    sortedInfo: {},
+    expandedRows: [],
```

Comment:
what do you think about expanding the first row by default?

Reply:
It depends from the list of files.
When the list is composed only by a folder, then it make sense, otherwise the user could see a folder already opened that is not supposed to be.

Reply:
ahh true - that makes sense. just a thought I had